### PR TITLE
v0.11 documentation

### PR DESCRIPTION
**Pull request by sriumcp**
Fixes the checked items in iter8-tools/iter8#1297 (see below)

- [x] Fixing landing page to remove "secure". Also add social (GitHub, slack, and twitter icons before conceived at)
- [ ] Describe use cases in landing page
- [ ] Update adopters
- [ ] Align CLI documentation with concepts
- [x] Replace your first experiment with the simplest k8s HTTP load test experiment with SLOs
- [x] Add twitter to Get Help section
- [x] Update installation
- [x] Reorganize tutorials as follows. Other pages (like kserve) currently under integrations will continue to be slotted under integrations.

```shell
.
├── custom-metrics
│   ├── one-app-version.md
│   └── two-or-more-app-versions.md
├── integrations
│   ├── local.md
│   └── github-actions.md
│   └── istio.md
├── load-test-grpc-with-slos.md
└── load-test-http-with-slos.md
```

- [x] Update concepts to help with descriptions in your first experiment and other tutorials
- [x] Under user guide/commands, update CLI documentation
- [x] Under user guide/topics, introduce the following structure

```shell
topics
├── tasks
│   ├── assess.md
│   ├── custommetrics.md
│   ├── grpc.md
│   ├── http.md
│   └── ready.md
├── metrics
├── namespace-and-group.md
└── experiment-runners.md
```

- [x] Update contributing
- [x] Update roadmap
- [x] Update community
- [ ] Fix all the README and contributing pages

### CODE REVIEW

File: docs/tutorials/integrations/ghactions.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ template: main.html
...
 
-Install the latest version of Iter8 using the GitHub Action `iter8-tools/iter8@v0.10`. Once installed, Iter8 can be used as documented (see [user guide](../../user-guide/commands/iter8.md)) in `run` actions. For example:
+Install the latest version of Iter8 CLI using the GitHub Action `iter8-tools/iter8@v0.11`. Once installed, Iter8 CLI can be used as documented in the [tutorials](../../getting-started/your-first-experiment.md) and [user guide](../../user-guide/commands/iter8.md). For example:
 
 ```yaml
-- uses: iter8-tools/iter8@v0.10 # install Iter8
+- uses: iter8-tools/iter8@v0.11 # install Iter8
 - run: |
     iter8 version
     iter8 launch -c load-test-http \
```
Comment (kalantar) [inline docs/tutorials/integrations/ghactions.md:13]: Suggest changing this to match the new approach:  `--set "tasks={http,assess}"` instead of `-c`.

**Comment by sriumcp** [comment]
The latest commit updates "your first experiment" to make it a Kubernetes experiment.

@kalantar The links in this experiment are broken (though the text is correct). Also, manually verified that the steps work as written.

So, you will need to try `mkdocs serve` instead of `mkdocs serve -s` after git cloning my fork and switching to my branch.

**Comment by sriumcp** [comment]
@kalantar Latest commit handles the following tutorials...

```shell
├── integrations
│   ├── local.md
│   └── github-actions.md
├── load-test-grpc-with-slos.md
└── load-test-http-with-slos.md
```

**Comment by sriumcp** [comment]
@kalantar @Alan-Cha Updated the custom metrics documentation in this PR (for the case of a single version; two version documentation is yet to come).

File: docs/tutorials/custom-metrics/one-version.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ```
+
+***
+
+## Launch experiment
+
+```shell
+iter8 k launch \
+--set "tasks={custommetrics,assess}" \
+--set custommetrics.templates.istio-prom="https://raw.githubusercontent.com/iter8-tools/iter8/master/custommetrics/istio-prom.tpl" \
```
[the remark this answers is not in the document]
Comment (sriumcp) [reply docs/tutorials/custom-metrics/one-version.md:40]: At the moment, only URLs are supported. 

The local file way of reading is lower priority, considering how easy it is to create a URL (e.g., using a gist on GitHub even without the need for a GitHub repo). Of course, if users push for it, we can reconsider.

**Comment by sriumcp** [comment]
@kalantar @Alan-Cha Updated the documentation for the `assess` and `ready` tasks.

**Comment by sriumcp** [comment]
Fixing `http` and `grpc` task descriptions.

File: docs/getting-started/concepts.md [hunk 3]
```diff
@@ -3,19 +3,33 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is a metrics-driven release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well, are secure, and maximize business value.
+Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well and maximize business value.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:6]: > Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams.

Do we want to say "**the** Kubernetes testing platform" or "**a** Kubernetes testing platform"?

File: docs/getting-started/concepts.md [hunk 4]
```diff
@@ -3,19 +3,33 @@ template: main.html
...
 
-## Iter8 experiment
+## Iter8 Experiment
 <p align='center'>
   <img alt-text="load-test-http" src="../../images/iter8-intro-dark.png" width="70%" />
 </p>
 
-Iter8 experiments make it simple to collect performance, risk, and business metrics for apps and ML models, assess, compare and validate one or more app/ML model versions, promote the winning version, and maximize business value in each release.
+Iter8 experiments makes it simple to collect performance and business metrics for apps and ML models, assess and compare one or more app/ML model versions, validate [service-level objectives (SLOs)](#service-level-objectives), promote the winning version, and maximize business value during each release.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:13]:
```suggestion
Iter8 experiments make it simple to collect performance and business metrics for apps and ML models, assess and compare one or more app/ML model versions, validate [service-level objectives (SLOs)](#service-level-objectives), promote the winning version, and maximize business value during each release.
```

File: docs/getting-started/concepts.md [hunk 5]
```diff
@@ -3,19 +3,33 @@ template: main.html
...
 
-## Experiment chart
-Iter8 experiment charts are [Helm charts](https://helm.sh/docs/topics/charts/) under the covers, and enable simple, declarative, and reusable metrics-driven experiments. Iter8 combines experiment charts with user supplied values to generate runnable `experiment.yaml` files or Kubernetes experiment manifests. The former is used for running experiments in the local environment, while the latter is used for running experiments inside Kubernetes.
+Specifying an Iter8 experiment involves specifying the list of [tasks](#tasks) executed during the experiment, their [parameters](../user-guide/topics/parameters.md), and the experiment [runner](#loops-and-runner).
+
+### Tasks
+An experiment is an ordered collection of tasks that are executed sequentially. Iter8 provides pre-defined tasks for various functions such as generating load and collecting Iter8's built-in metrics for HTTP and gRPC services, collecting custom metrics for one or more versions of an app from databases, assessing [SLOs](#service-level-objectives), and checking if the application is ready.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:18]:
```suggestion
An experiment is an ordered collection of tasks that are executed sequentially. Iter8 provides pre-defined tasks for various functions such as generating load and collecting Iter8's built-in metrics for HTTP and gRPC services, collecting custom metrics from databases, assessing [SLOs](#service-level-objectives), and checking if the application is ready.
```

This is a pretty long sentence so I was wondering if we could shorten it a bit.

File: docs/getting-started/concepts.md [hunk 6]
```diff
@@ -3,19 +3,33 @@ template: main.html
...
+
+### Loops
+A single loop of an experiment involves each task in the experiment executing once. Iter8 experiments can be set up to loop once and finish (single-loop), or loop repeatedly over time (multi-loop).
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:21]:
```suggestion
A basic experiment involves each task in the experiment executing once but Iter8 experiments can also loop, meaning they will executing tasks more than once. You can configure experiments to loop once and finish (single-loop), or loop repeatedly over time (multi-loop).
```

File: docs/getting-started/concepts.md [hunk 7]
```diff
@@ -3,19 +3,33 @@ template: main.html
...
+
+### Execution Environments
+Experiments can run inside Kubernetes clusters, in the local environment, and within CI/CD/GitOps workflows including GitHub Actions workflows.
+
+#### Runners
+A single-loop experiment that is run inside a Kubernetes cluster uses the Kubernetes [job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) workload as its runner. A multi-loop experiment that is run inside a Kubernetes cluster uses the Kubernetes [cronjob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) as its runner.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:27]: Should this section be somehow combined with the loops section?

File: docs/user-guide/commands/iter8_gen.md [hunk 8]
```diff
@@ -11,11 +11,13 @@ Generate experiment.yaml file by combining an experiment chart with values
...
 
 
-Generate an experiment.yaml file by combining an experiment chart with values.
+Generate an experiment.yaml file by combining the Iter8 experiment chart with values.
 
-    $ iter8 gen --sourceDir /path/to/load-test-http --set url=https://httpbin.org/get
+	iter8 gen --set "tasks={http}" --set http.url=https://httpbin.org/get
 
-This command is intended for development and testing of experiment charts. For production usage, the launch command is recommended.
+This command supports setting values using the same mechanisms as in Helm. Please see  https://helm.sh/docs/chart_template_guide/values_files/ for more detailed descriptions. In particular, this command supports the --set, --set-file, --set-string, and -f (--values) options all of which have the same behavior as in Helm.
```
Comment (Alan-Cha) [inline docs/user-guide/commands/iter8_gen.md:18]:
```suggestion
This command supports setting values using the same mechanisms as in Helm. Please see [https://helm.sh/docs/chart_template_guide/values_files/](https://helm.sh/docs/chart_template_guide/values_files/) for more detailed descriptions. In particular, this command supports the `--set`, `--set-file`, `--set-string`, and `-f` (`--values`) options all of which have the same behavior as in Helm.	
```

File: docs/user-guide/commands/iter8_k_launch.md [hunk 9]
```diff
@@ -6,26 +6,26 @@ hide:
...
+	--set runner=job \
+	--dry
 
-By default, the current directory is used to download and unpack the experiment chart. Change this location using the destDir option.
+The launch command creates the 'charts' subdirectory under the current working directory, downloads the Iter8 experiment chart, and places it under 'charts'. This behavior can be controlled using various launch flags.
 
-	$ iter8 k launch -c load-test-http \
-	  --set url=https://httpbin.org/get \
-	  --destDir /tmp
+This command supports setting values using the same mechanisms as in Helm. Please see  https://helm.sh/docs/chart_template_guide/values_files/ for more detailed descriptions. In particular, this command supports the --set, --set-file, --set-string, and -f (--values) options all of which have the same behavior as in Helm.
```
Comment (Alan-Cha) [inline docs/user-guide/commands/iter8_k_launch.md:28]:
```suggestion
This command supports setting values using the same mechanisms as in Helm. Please see [https://helm.sh/docs/chart_template_guide/values_files/](https://helm.sh/docs/chart_template_guide/values_files/) for more detailed descriptions. In particular, this command supports the `--set`, `--set-file`, `--set-string`, and `-f` (`--values`) options all of which have the same behavior as in Helm.
```

File: docs/user-guide/commands/iter8_launch.md [hunk 10]
```diff
@@ -6,26 +6,25 @@ hide:
...
+	--set http.url=https://httpbin.org/get \
+	--dry
 
-By default, the current directory is used to download and unpack the experiment chart. Change this location using the destDir option.
+The launch command creates the 'charts' subdirectory under the current working directory, downloads the Iter8 experiment chart, and places it under 'charts'. This behavior can be controlled using various launch flags.
 
-	$ iter8 launch -c load-test-http \
-	  --set url=https://httpbin.org/get \
-	  --destDir /tmp
+This command supports setting values using the same mechanisms as in Helm. Please see  https://helm.sh/docs/chart_template_guide/values_files/ for more detailed descriptions. In particular, this command supports the --set, --set-file, --set-string, and -f (--values) options all of which have the same behavior as in Helm.
```
Comment (Alan-Cha) [inline docs/user-guide/commands/iter8_launch.md:27]:
```suggestion
This command supports setting values using the same mechanisms as in Helm. Please see [https://helm.sh/docs/chart_template_guide/values_files/](https://helm.sh/docs/chart_template_guide/values_files/) for more detailed descriptions. In particular, this command supports the `--set`, `--set-file`, `--set-string`, and `-f` (`--values`) options all of which have the same behavior as in Helm.
```

File: docs/user-guide/commands/iter8_run.md [hunk 11]
```diff
@@ -11,9 +11,9 @@ Run an experiment
 ### Synopsis
 
 
-Run an experiment specified in experiment.yaml output result to result.yaml.
+Run an experiment specified in experiment.yaml. This command modifies the experiment.yaml file as part of the run.
```
Comment (Alan-Cha) [inline docs/user-guide/commands/iter8_run.md:14]:
```suggestion
Run an experiment specified in `experiment.yaml`. This command modifies the `experiment.yaml` file as part of the run.
```

File: docs/user-guide/commands/iter8_run.md [hunk 12]
```diff
@@ -11,9 +11,9 @@ Run an experiment
...
 
-	$ iter8 run
+	iter8 run
 
 This command is intended for development and testing of experiment charts and tasks. For production usage, the iter8 launch command is recommended.
```
Comment (Alan-Cha) [inline docs/user-guide/commands/iter8_run.md:18]:
```suggestion
This command is intended for development and testing of experiment charts and tasks. For production usage, the `iter8 launch` command is recommended.
```

File: docs/getting-started/concepts.md (hunk 3 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: I prefer **the** ; **the one and only** is really I want to see but that is a bit too cheesy :)

File: docs/getting-started/your-first-experiment.md [hunk 13]
```diff
@@ -4,28 +4,92 @@ template: main.html
...
---8<-- "docs/getting-started/installiter8cli.md"
+## Install Iter8 CLI
+--8<-- "docs/getting-started/installbrewbins.md"
+
+***
 
-## 2. Launch experiment
-Use `iter8 launch` to benchmark the HTTP service whose URL is https://httpbin.org/get.
+## Launch experiment
+Use the [iter8 k launch]() command to launch the Iter8 experiment inside the Kubernetes cluster.
```
[the remark this answers is not in the document]
Comment (sriumcp) [reply docs/getting-started/your-first-experiment.md]: Fixed.

File: docs/getting-started/concepts.md (hunk 4 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

File: docs/getting-started/concepts.md (hunk 6 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

File: docs/getting-started/concepts.md (hunk 5 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

File: docs/getting-started/installghaction.md [hunk 14]
```diff
@@ -0,0 +1,6 @@
+=== "GitHub Actions"
+    Install the latest stable release of the Iter8 CLI in your GitHub Actions workflow as follows.
+
+    ```shell
```
[the remark this answers is not in the document]
Comment (sriumcp) [reply docs/getting-started/installghaction.md:4]: Fixed.

File: docs/user-guide/commands/iter8_gen.md (hunk 8 above)
Comment (sriumcp) [reply docs/user-guide/commands/iter8_gen.md:18]: iter8 command documentation is auto generated (using `iter8 gen`). So, we will need to fix this not directly in this repo but in the iter8 repo.

File: docs/user-guide/commands/iter8_launch.md (hunk 10 above)
Comment (sriumcp) [reply docs/user-guide/commands/iter8_launch.md:27]: To be fixed in the `iter8` repo.

File: docs/user-guide/commands/iter8_run.md (hunk 11 above)
Comment (sriumcp) [reply docs/user-guide/commands/iter8_run.md:14]: To be fixed in the iter8 repo.

File: docs/user-guide/commands/iter8_run.md (hunk 12 above)
Comment (sriumcp) [reply docs/user-guide/commands/iter8_run.md:18]: To be fixed in the `iter8` repo.

File: docs/getting-started/concepts.md [hunk 15]
```diff
@@ -3,19 +3,40 @@ template: main.html
...
+Iter8 experiments make it simple to collect performance and business metrics for apps and ML models, assess and compare one or more app/ML model versions, validate [service-level objectives (SLOs)](#service-level-objectives), promote the winning version, and maximize business value during each release.
 
-## Experiment chart
-Iter8 experiment charts are [Helm charts](https://helm.sh/docs/topics/charts/) under the covers, and enable simple, declarative, and reusable metrics-driven experiments. Iter8 combines experiment charts with user supplied values to generate runnable `experiment.yaml` files or Kubernetes experiment manifests. The former is used for running experiments in the local environment, while the latter is used for running experiments inside Kubernetes.
+### Tasks
+An experiment is a set of tasks that are executed in a specific sequence. Iter8 provides pre-defined tasks for various functions such as the following:
+
+1. Generating load and collecting Iter8's built-in metrics for HTTP services.
+2. Generating load and collecting Iter8's built-in metrics for gRPC services.
+3. Fetching metrics for one or more versions of an app from databases.
```
Comment (kalantar) [inline docs/getting-started/concepts.md:20]: I wonder if we can/should be more explicit: from metrics databases.  _or_ from REST accessible metrics stores.

File: docs/getting-started/concepts.md [hunk 16]
```diff
@@ -3,19 +3,40 @@ template: main.html
...
+4. Assessing [SLOs](#service-level-objectives) for app versions.
+5. Checking if app is ready.
+
+### Loops
+A single loop of an experiment involves the entire sequence of [tasks](#tasks) in the experiment executing once. You can configure experiments to loop once and finish (**single-loop**), or loop repeatedly over time (**multi-loop**).
+
+### Execution environments
+Experiments can run inside Kubernetes clusters, in the local environment, and within CI/CD/GitOps workflows including GitHub Actions workflows. Experiments that are executed inside Kubernetes clusters are referred to as **Kubernetes experiments**. All other experiments are referred to as **local experiments**.
+
+### Runner
```
Comment (kalantar) [inline docs/getting-started/concepts.md:30]: Perhaps we should title this "Kubernetes Runner"; currently this is kubernetes centric.

File: docs/getting-started/concepts.md (hunk 15 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

**Review by kalantar — APPROVED** [review]

File: README.md [hunk 17]
```diff
@@ -1,10 +1,10 @@
-# Iter8: Metrics-Driven Release Optimizer
+# Iter8: Kubernetes Release Optimizer
 
 [![Iter8 release](https://img.shields.io/github/v/release/iter8-tools/iter8?sort=semver)](https://github.com/iter8-tools/iter8/releases)
 [![GoDoc](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)](https://pkg.go.dev/github.com/iter8-tools/iter8)
 ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/iter8-tools/iter8/tests?label=Unit%20tests)
 
-> Iter8 is a metrics-driven release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well, are secure, and maximize business value.
+> Iter8 is a Kubernetes release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that Kubernetes apps and ML models perform well and maximize business value.
```
Comment (Alan-Cha) [inline README.md:7]: This says "a Kubernetes release optimizer"...

File: docs/getting-started/concepts.md [hunk 18]
```diff
@@ -3,32 +3,39 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well and maximize business value.
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that Kubernetes apps and ML models perform well and maximize business value.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:6]: ... whereas this says "a Kubernetes release optimizer".

Comment (Alan-Cha) [reply docs/getting-started/concepts.md:6]: ... whereas this says "DevOps".

**Review by kalantar — APPROVED** [review]

**Pull request merged**